### ISSUE TITLE
An exception has been thrown during the rendering of a template

### ISSUE THREAD

**Issue by berecont** [pdir/contao-seo-plugin#1]
lokale 5.3.11 (laragon) - danach update auf 5.4.0-RC2
Installation von pdir/contao-seo-plugin
Login ins Backend:

`An exception has been thrown during the rendering of a template ("Pdir\ContaoWebtoolsBundle\EventListener\BackendMenuListener::__construct(): Argument #1 ($security) must be of type Symfony\Component\Security\Core\Security, Symfony\Bundle\SecurityBundle\Security given, called in C:\laragon\www\dev54\var\cache\dev\ContainerYxDiymE\getPdir_ContaoWebtools_Listener_Menu_BackendService.php on line 34").`

```
Twig\Error\RuntimeError:
An exception has been thrown during the rendering of a template ("Pdir\ContaoWebtoolsBundle\EventListener\BackendMenuListener::__construct(): Argument #1 ($security) must be of type Symfony\Component\Security\Core\Security, Symfony\Bundle\SecurityBundle\Security given, called in C:\laragon\www\dev54\var\cache\dev\ContainerYxDiymE\getPdir_ContaoWebtools_Listener_Menu_BackendService.php on line 34").

  at C:\laragon\www\dev54\vendor\contao\core-bundle\templates\Backend\be_menu.html.twig:13
  at Twig\Template->yield(array('contao' => object(ContaoVariable), 'request_token' => object(Stringable@anonymous), 'app' => object(AppVariable)))
     (C:\laragon\www\dev54\vendor\twig\twig\src\Template.php:335)
  at Twig\Template->render(array())
     (C:\laragon\www\dev54\vendor\twig\twig\src\TemplateWrapper.php:38)
  at Twig\TemplateWrapper->render(array())
     (C:\laragon\www\dev54\vendor\twig\twig\src\Environment.php:307)
  at Twig\Environment->render('@ContaoCore/Backend/be_menu.html.twig')
     (C:\laragon\www\dev54\vendor\contao\core-bundle\contao\controllers\BackendMain.php:249)
  at Contao\BackendMain->compileTemplateData(array('version' => 'Version 5.4.0-RC2', 'main' => '<!-- TEMPLATE START: vendor/contao/core-bundle/contao/templates/backend/be_welcome.html5 --><div id="tl_soverview">  <div id="tl_messages">    <h2>Hinweise</h2>    <p>Dies scheint Ihre erste Anmeldung zu sein. Willkommen bei Contao!</p>    <p class="tl_error">Es ist keine Administrator-E-Mail-Adresse in den <a href="/contao?do=settings&ref=WIj_zGns" target="_top">System-Einstellungen</a> hinterlegt.</p>  </div>  <div id="tl_shortcuts">    <h2>Backend-Tastaturkürzel</h2>    <p>Wie Sie Ihren Arbeitsablauf durch die Verwendung von <a href="https://to.contao.org/docs/shortcuts" target="_blank" rel="noreferrer noopener">Tastaturkürzeln</a> beschleunigen.</p>  </div>  <div id="tl_versions">    <h2>Zuletzt bearbeitet</h2>          <p>Zurzeit sind keine Versionen vorhanden.</p>      </div></div><!-- TEMPLATE END: vendor/contao/core-bundle/contao/templates/backend/be_welcome.html5 -->', 'title' => 'Dashboard', 'headline' => 'Dashboard', 'theme' => 'flexible', 'language' => 'de', 'host' => 'dev54.local', 'charset' => 'UTF-8', 'home' => 'Startseite', 'isPopup' => null, 'learnMore' => 'Mehr erfahren auf <a href="https://contao.org" target="_blank" rel="noreferrer noopener">contao.org</a>'))
     (C:\laragon\www\dev54\vendor\contao\core-bundle\contao\controllers\BackendMain.php:207)
  at Contao\BackendMain->output()
     (C:\laragon\www\dev54\vendor\contao\core-bundle\contao\controllers\BackendMain.php:150)
  at Contao\BackendMain->run()
     (C:\laragon\www\dev54\vendor\contao\core-bundle\src\Controller\BackendController.php:44)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (C:\laragon\www\dev54\vendor\symfony\http-kernel\HttpKernel.php:181)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (C:\laragon\www\dev54\vendor\symfony\http-kernel\HttpKernel.php:76)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (C:\laragon\www\dev54\vendor\symfony\http-kernel\Kernel.php:197)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (C:\laragon\www\dev54\public\index.php:42)
```

composer.json:
```
{
    "type": "project",
    "require": {
        "contao/calendar-bundle": "^5.4",
        "contao/comments-bundle": "^5.4",
        "contao/conflicts": "*@dev",
        "contao/faq-bundle": "^5.4",
        "contao/listing-bundle": "^5.4",
        "contao/manager-bundle": "^5.4",
        "contao/news-bundle": "^5.4",
        "contao/newsletter-bundle": "^5.4",
        "pdir/contao-seo-plugin": "^1.0"
    },
    "minimum-stability": "RC",
    "prefer-stable": true,
    "config": {
        "allow-plugins": {
            "composer/package-versions-deprecated": true,
            "contao-community-alliance/composer-plugin": true,
            "contao-components/installer": true,
            "contao/manager-plugin": true,
            "php-http/discovery": true
        }
    },    
    "extra": {
        "public-dir": "public",
        "contao-component-dir": "assets"
    },
    "scripts": {
        "post-install-cmd": [
            "@php vendor/bin/contao-setup"
        ],
        "post-update-cmd": [
            "@php vendor/bin/contao-setup"
        ]
    }
}
```

**Comment by fritzmg** [comment]
This is an issue of https://github.com/pdir/contao-webtools

To stay compatible with 4.13 and 5.x, https://github.com/pdir/contao-webtools needs to change to the `TokenStorageInterface` and get the user object from there (via `$this->tokenStorage->getToken()?->getUser()`).

**Comment by MDevster** [comment]
Fixed in version [1.6.0](https://github.com/pdir/contao-webtools/releases/tag/1.6.0).
